server/service/src/service_provider.rs

```diff
@@ -63,6 +63,12 @@ use repository::{
     PaginationOption, RepositoryError, StorageConnection, StorageConnectionManager, Store,
     StoreFilter, StoreSort,
 };
+use rust_embed::RustEmbed;
+
+#[derive(RustEmbed)]
+// Relative to server/Cargo.toml
+#[folder = "../../client/packages/host/dist"]
```

Comment:
I think you can point directly to the locale directory, and can also do cfg flag to point to translations src in for dev `#[cfg(debug_assertions)]` and `#[not(cfg(debug_assertions))]`